setup.py

```diff
@@ -15,17 +16,24 @@
 ]
 
 __keywords__ = ["Echarts", "charts", "plotting-tool"]
-
 # Load the package's _version.py module as a dictionary.
 here = os.path.abspath(os.path.dirname(__file__))
 about = {}
 with open(os.path.join(here, __title__, "_version.py")) as f:
     exec(f.read(), about)
 
+
+def read(afile):
+    """Read a file into setup"""
+    with codecs.open(afile, 'r', 'utf-8') as opened_file:
+        return opened_file.read()
+
+
 setup(
     name=__title__,
     version=about["__version__"],
     description=__description__,
+    long_description=read("README.md"),
```

Comment:
This does not work.Markdown support is a new feature for PyPi in last two month.You may follow [this article](https://dustingram.com/articles/2018/03/16/markdown-descriptions-on-pypi) now.

Reply:
*README.md* is too long. It is better use another *long_description.md* file.

Reply:
not long enough! :), I have not included change log inside it yet.